### PR TITLE
docs: Lilac release notes

### LINKED ISSUE #15: Warn about courseware_studentmodule index addition in Lilac release notes
We should incorporate a warning and instructions about the following PR specifically in the release notes, including the fake migration idea:

https://github.com/edx/edx-platform/pull/26617

### LINKED ISSUE #61: Lilac release notes
The wiki page for [Process to Create an Open edX Release](https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release) has a couple of steps for creating release notes:

- [x] Get release notes started
- [x] Make release-specific pages for installation and release notes

In addition, we need to:

- [x] add an appropriate TOC to the release notes
- [x] add any items from the [Lilac wiki page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2023915819/Lilac)  
- [x] Review and incorporate any feature posts from [edX Partner announcements](https://partners.edx.org/announcements)
- [ ] Review and incorporate any feature posts from the [open edX blog](https://open.edx.org/category/blog/)
- [ ] Compile a list of relevant repos for review, e.g. ORA, Proctoring, etc. 
- [ ] Review commit messages from relevant repos 
- [ ] Review [DEPR tickets ](https://openedx.atlassian.net/secure/RapidBoard.jspa?rapidView=452)

### PR DESCRIPTION
**Pull request by pdpinch**
## [OSPR-5766](https://openedx.atlassian.net/browse/OSPR-5766)

DRAFT of release notes for lilac open-release

fixes https://github.com/openedx/build-test-release-wg/issues/61
fixes https://github.com/openedx/build-test-release-wg/issues/15  

### Date Needed (optional)

5/9/2020

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @nedbat 
- [ ] Subject matter expert: @michellephilbrick @marcotuts 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review: @marcotuts 
- [ ] Partner support: none
- [ ] PM review: none

FYI: @arbrandes @BbrSofiane @mkdavies @briangrossman

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [x] Build an RTD draft for your branch and add a link here: https://draft-lilac-release-notes.readthedocs.io/en/latest/lilac.html

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pdpinch! I've created [OSPR-5766](https://openedx.atlassian.net/browse/OSPR-5766) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by pdpinch** [comment]
This is an early draft of the release notes. At this time, I'm mostly interested in feedback on the tables of contents and high-level formatting. 

Regarding content, I would consider this the bare minimum. My plan is to include:
- [x] items from the [Lilac wiki page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2023915819/Lilac)  
- [x] MFEs
- [x] feature posts from [edX Partner announcements](https://partners.edx.org/announcements)
- [x] feature posts from the [open edX blog](https://open.edx.org/category/blog/)
- [ ] features from relevant repos e.g. ORA, Proctoring, etc. 
- [ ] [DEPR tickets ](https://openedx.atlassian.net/secure/RapidBoard.jspa?rapidView=452)

**Comment by nedbat** [comment]
@pdpinch Thanks for taking the lead on this!  I have asked @michellephilbrick to get product info together also as raw materials.

File: en_us/open_edx_release_notes/source/lilac.rst [hunk 1]
```diff
@@ -4,7 +4,150 @@
...
+
+.. contents::
+ :depth: 1
+ :local:
+
+===================
+Learner Experiences
+===================
+
+Account MFE
```
Comment (bradenmacdonald) [inline en_us/open_edx_release_notes/source/lilac.rst]: Suggestion: Can we define "MFE" somewhere near the beginning of this document? Some people may not know what it means, and google doesn't define it unless you add in other keywords like "frontend".

**Comment by natabene** [comment]
@pdpinch Thanks for starting this.

File: en_us/open_edx_release_notes/source/lilac.rst (hunk 1 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/lilac.rst]: Even better: just say "micro-frontend" instead :)

File: en_us/open_edx_release_notes/source/lilac.rst [hunk 2]
```diff
@@ -4,7 +4,150 @@
 Open edX Lilac Release
 ######################
 
-This page will list the highlights of the Lilac release of the Open edX platform.
+These are the release notes for the Lilac release, the 12th community release of the Open edX Platform, spanning changes from November 12 2020 to April 9 2021.  You can also review details about `earlier releases`_ or learn more about the `Open edX Platform`_ if you are new to Open edX.
```
Comment (nedbat) [inline en_us/open_edx_release_notes/source/lilac.rst]:
```suggestion
These are the release notes for the Lilac release, the 12th community release of the Open edX Platform, spanning changes from November 12 2020 to April 9 2021.  You can also review details about `earlier releases`_ or learn more about the `Open edX Platform`_.
```

**Comment by nedbat** [comment]
I've created a readthedocs project for this branch: https://draft-lilac-release-notes.readthedocs.io/en/latest/
But I think it needs a webhook installed in the mitodl fork in order to update automatically.

**Comment by pdpinch** [comment]
Thanks @nedbat -- I think maybe the easiest way to do that is give you access to my fork so you can add the webhook? If you disagree, just let me know how to set up the webhook and I'll do it.

**Comment by nedbat** [comment]
All set: webhook is added.

File: en_us/open_edx_release_notes/source/lilac.rst (hunk 1 above)
Comment (pdpinch) [reply en_us/open_edx_release_notes/source/lilac.rst]: I spelled out Micro Frontend. @bradenmacdonald let me know if that's sufficient.

Comment (bradenmacdonald) [reply en_us/open_edx_release_notes/source/lilac.rst]: Works for me - thanks!

**Comment by pdpinch** [comment]
I added what I could extract from the edX Partner announcements, but I have questions. I'll tag some folks.

**Comment by michellephilbrick** [comment]
@pdpinch The Product team has finished providing their updates to this doc: https://docs.google.com/document/d/1ZD8rCrBHO6rg-a08ZUEvW_iX-2m7HBiCK09Gr-Rg_uc/edit?userstoinvite=wkhalid%40edx.org&ts=60afb481&actionButton=1#heading=h.x7y6fix3ys2z

Please let me know if you need anything additional at this time.

File: en_us/open_edx_release_notes/source/lilac.rst [hunk 3]
```diff
@@ -4,7 +4,186 @@
...
+===================
+Learner Experiences
+===================
+
+Open-Response Assessments
+-------------------------
+- Learners can submit rich text responses
+- Grading status message
+- Allow viewing ORA steps after peer review
+- See the Authoring Experience section below for more ORA enhancements
```
Comment (pdpinch) [inline en_us/open_edx_release_notes/source/lilac.rst:25]: @marcotuts can you confirm these ORA enhancements for learners are include in Lilac? The text is my interpretation of https://partners.edx.org/announcements/open-response-assessment-update which isn't available to the community.

File: en_us/open_edx_release_notes/source/lilac.rst [hunk 4]
```diff
@@ -4,7 +4,186 @@
...
+
+Open-Response Assessments
+-------------------------
+
+- Added username details to ORA report download
+- Added problem name and location to ORA report download
+- Added ORA zipped file download for submission text + attached files
+- Open response assessment problems as a component category
+- Separate assessment steps & schedule authoring areas in Studio
+- See the Learner Experience section above for more ORA enhancements
```
Comment (pdpinch) [inline en_us/open_edx_release_notes/source/lilac.rst:104]: @marcotuts same question, for instructors. Can you confirm these ORA enhancements are included in Lilac? Let me know if I need any copy editing.

**Comment by pdpinch** [comment]
Thanks @michellephilbrick. I think I've incorporated everything from the product team. Please ask them to take a look at https://draft-lilac-release-notes.readthedocs.io/en/latest/lilac.html to see the current draft.

**Comment by openedx-webhooks** [comment]
@pdpinch 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-documentation&entry.867055334=https://github.com/edx/edx-documentation/pull/1939&entry.1484655318=https://github.com/pdpinch&entry.752974735=2021-05-01+16:49&entry.1917517419=2021-06-07+19:56&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by nedbat** [comment]
I've cherry-picked this onto Lilac.

**Comment by michellephilbrick** [comment]
> Thanks @michellephilbrick. I think I've incorporated everything from the product team. Please ask them to take a look at https://draft-lilac-release-notes.readthedocs.io/en/latest/lilac.html to see the current draft.

@pdpinch no further comments or feedback from the Product team on the draft!

**Pull request merged**